### ISSUE TITLE
[c-sharp-operations] operation name differs in casing with operation name from GraphQL query

### ISSUE THREAD

**Issue by cklam2** [dotansimha/graphql-code-generator#4260]
**Describe the bug**
The operation name added to the C# call differs in casing from the GraphQL operation. GraphQL servers that work case sensitive will complain about non existing operation name. 

**To Reproduce**
Use the live [GraphQL code generator](https://graphql-code-generator.com/) and enter the following blocks. Then inspect its output.

1. My GraphQL schema:

```graphql
type query {
  me: Int!
}
```

2. My GraphQL operations:

```graphql
query findUser {
  me 
}
```

3. My `codegen.yml` config file:

```yml
generates:
  src/main/c-sharp/my-org/my-app/Operations.cs:
    plugins:
      - c-sharp-operations
```

**Expected behavior**
Plugin should generate this:
```C#
public class FindUserGQL{
  public static GraphQLRequest getFindUserGQL() {
    return new GraphQLRequest {
      Query = FindUserDocument,
      OperationName = "findUser"
    };
  }
      
  public static string FindUserDocument = @"
    query findUser {
      me
    }";
}
```

Instead it generates this:
```C#
  ...
  Query = FindUserDocument,
  OperationName = "FindUser"
  ...
}
```
As can be seen the value from OperationName (FindUser) is not the same as the one from GraphQL query (findUser).

**Environment:**
Can be reproduced in the live [GraphQL code generator](https://graphql-code-generator.com/)

**Additional context**
- Server is a GraphQL Apollo server running on Linux

**Comment by cklam2** [comment]
Code seems to convert the operation name to either PascalCase or SentenceCase (default behavior of convertName). Eg
```findUser => FindUser```
but
```FINDUSER => Finduser```

So workaround for now is to stick to either PascalCase (FindUser) or Sentencecase (Finduser) for operation names in graphql query.

**Comment by cklam2** [comment]
Solved with #4264

**Comment by dotansimha** [comment]
Fixed in `v1.16.0`
